apps/camera/js/lib/camera/camera.js

```diff
@@ -1251,8 +1307,13 @@ Camera.prototype.isMode = function(mode) {
  *
  * @private
  */
-Camera.prototype.startVideoTimer = function() {
-  this.set('videoStart', new Date().getTime());
+Camera.prototype.startVideoTimer = function(resume) {
+ if (resume) {
+    var delta = this.videoStopped - this.get('videoStart');
+    this.set('videoStart', (new Date().getTime()) - delta);
```

Comment:
Also, when you change this to `Date.now()`, the extra set of parenthesis are not needed :-)